gradle/versions.gradle

```diff
@@ -45,7 +50,7 @@ dependencyManagement {
 
     dependency 'com.google.guava:guava:31.1-jre'
 
-    dependency 'com.graphql-java:graphql-java:18.1'
+    dependency 'com.graphql-java:graphql-java:19.2'
```

Comment:
Is the graphql version change required as part of the tuweni upgrade? Can it be put in a different PR?

Reply:
It is required, but could be done separate. This old version apparently depends on an old version of antlr which has a vulnerability. Tuweni upgraded, and now Besu complains that the new version of antlr passed in transitively is not compatible with the antlr schema used by graphql.

By upgrading graphql, we remove entirely its dependency to antlr.

Reply:
Sounds good, just wanting to make sure there was an intentional reason that was on record.